### PR TITLE
Testing Coverage Badge plus additional tests

### PR DESCRIPTION
**Pull request by P4T12ICK**
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

DNS Query Length With High Standard Deviation :white_check_mark:
- diskUsage: 200704
- runDuration: 1.5150000000000001s
- scanCount: 1627

### CODE REVIEW

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Malicious PowerShell Process With Obfuscation Techniques :white_check_mark:
- diskUsage: 356352
- runDuration: 18.649s
- scanCount: 29687

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Malicious PowerShell Process - Execution Policy Bypass :white_check_mark:
- diskUsage: 438272
- runDuration: 1.698s
- scanCount: 929

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

First Time Seen Running Windows Service :x:
- diskUsage: 102400
- runDuration: 0.302s
- scanCount: 649

File: detections/cloud/high_number_of_login_failures_from_a_single_source.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: High Number of Login Failures from a single source
 id: 7f398cfb-918d-41f4-8db8-2e2474e02222
 version: 1
 date: '2020-12-16'
-description: This search will detect more than 5 login failures in Office365 Azure Active Directory from a single source IP address. Please adjust the threshold value of 5 as suited for your environment.
+description: This search will detect more than 5 login failures in Office365 Azure Active Directory from a single source IP address. Please adjust the threshold value of 5 as suited for your environment. Deprecated because duplicate.
```
Comment (patel-bhavin) [inline detections/cloud/high_number_of_login_failures_from_a_single_source.yml]: @P4T12ICK : this search is not completely a duplicate. This search explicitly looks for 5 login failures happening from a particular src_ip

**Review by patel-bhavin — CHANGES_REQUESTED** [review]

File: detections/cloud/high_number_of_login_failures_from_a_single_source.yml (hunk 1 above)
Comment (patel-bhavin) [reply detections/cloud/high_number_of_login_failures_from_a_single_source.yml]: I believe the other one `O365 Excessive Authentication Failures Alert` is looking for multiple login failures by user.

File: detections/endpoint/malicious_powershell_process_with_obfuscation_techniques.yml [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+version: 4
+date: '2021-01-19'
 description: This search looks for PowerShell processes launched with arguments that
   have characters indicative of obfuscation on the command-line.
 how_to_implement: You must be ingesting data that records process activity from your
   hosts to populate the Endpoint data model in the Processes node. You must also be
   ingesting logs with both the process name and command line from your endpoints.
   The command-line arguments are mapped to the "process" field in the Endpoint data
-  model. Deprecated because the detection need rework and would need to work base on powershell logs.
+  model. Deprecated because the detection need rework and would need to work base
```
Comment (patel-bhavin) [inline detections/endpoint/malicious_powershell_process_with_obfuscation_techniques.yml]: we should probably clean this up

File: detections/cloud/high_number_of_login_failures_from_a_single_source.yml (hunk 1 above)
Comment (P4T12ICK) [reply detections/cloud/high_number_of_login_failures_from_a_single_source.yml]: fixed this one

File: detections/endpoint/malicious_powershell_process_with_obfuscation_techniques.yml (hunk 2 above)
Comment (P4T12ICK) [reply detections/endpoint/malicious_powershell_process_with_obfuscation_techniques.yml]: should be resolved

**Review by patel-bhavin — APPROVED** [review]

**Pull request merged**